flake.nix

```diff
@@ -4,6 +4,20 @@
   inputs.nixpkgs.url = github:NixOS/nixpkgs/nixos-20.09;
 
   outputs = { self, nixpkgs, ... }: {
+        dockerContainer.x86_64-linux = let
```

Comment:
Can you try `packages.x86_64-linux.dockerContainer` instead? This might fix the output of `nix flake show`.

Also, plz move it below `defaultPackage` :)